packages/dapp/src/components/Navbar/Menu/WalletMenu/WalletMenu.tsx

```diff
@@ -94,15 +99,21 @@ export const WalletMenu = ({ handleClose }: NavbarMenuProps) => {
       pageload: true,
       disableTrackingTool: [EventTrackingTool.ARCx, EventTrackingTool.Raleon],
     });
+    account.chainId &&
+      openInNewTab(
+        `${
+          getChainById(account.chainId).metamask.blockExplorerUrls[0]
+        }address/${account.address}`,
```

Comment:
Let's extract this to a new variable and then use the variable here instead of forming a string in the function argument

Reply:
Thanks Abhishek! Can you check if that would be suitable the way I changed it now?
https://github.com/lifinance/jumper.exchange/pull/631/commits/4a712c6d859b149e5c3ec1f6f2eeebbc9156aae3